doc/api-extensions.md

```diff
@@ -219,12 +219,12 @@ Introduces the ability to rename a volume group by setting "storage.lvm.vg\_name
 Introduces the ability to rename a thinpool name by setting "storage.thinpool\_name".
 
 ## network\_vlan
-This adds a new "vlan" property to "macvlan" network devices.
+This adds a new "vlan" property to "macvlan" and "physical" network devices.
 
 When set, this will instruct LXD to attach to the specified VLAN. LXD
 will look for an existing interface for that VLAN on the host. If one
 can't be found it will create one itself and then use that as the
-macvlan parent.
+macvlan parent if "macvlan" nictype is selected or as it if "physical" nictype is selected.
```

Comment:
Again, I never asked about adding a new property, just a new API extension string. The two are completely separate things!

The API extension strings are used to track when a given feature was added to LXD and doesn't have to be tied at all to a new property being added. It just means that LXD supports something new, which in this case is the combination of nictype=physical and the vlan property.

So don't add any new properties, just add "network_vlan_physical" to the list of strings in lxd/api_1_0.go and to the list in doc/api-extensions.md. That's it.